src/Workspaces/Core/Portable/LanguageServices/SyntaxFactsService/ISyntaxFactsService.cs

```diff
@@ -341,6 +341,8 @@ internal interface ISyntaxFactsService : ILanguageService
         Location GetDeconstructionReferenceLocation(SyntaxNode node);
 
         SyntaxToken? GetDeclarationIdentifierIfOverride(SyntaxToken token);
+
+        bool IdentifiesLambda(SyntaxToken token);
```

Comment:
on the fence if this is the right location for this method.

Reply:
I'm planning to leave as-is unless I hear some better suggestion ;-)

---
In reply to: [159457900](https://github.com/dotnet/roslyn/pull/23970#discussion_r159457900) [](ancestors = 159457900)